src/utils/index.ts

```diff
@@ -33,3 +34,7 @@ export const isValidURL = (url: string) => {
     return false
   }
 }
+
+export const isAnonFeedbackEnabled = () => {
+  return process.env.TESTIMONIALS === "ANON"
+}
```

Comment:
There are basically three values. I would recommend we store this information inside an enum instead. 

```ts
enum TestimonialDisplay {Hide, Show, Anon}
```

Reply:
@tusharmath , this check is only for the ANON, for ON and OFF we just have to pass it in the env and it will be handled